website/templates/project/view_file.mako

```diff
@@ -15,6 +15,7 @@
     <h2 class="break-word">
       ## Split file name into two parts: with and without extension
       ${file_name_title | h}<span id="file-ext">${file_name_ext | h}</span>
+      <a id="version-link"><span id="version-number"></span></a>
```

Comment:
This seems a little odd and unlike the way we're doing things elsewhere in the file (such as right above and right below), but I'm not positive because I'm not big into mithril and am not sure if this is proper.